dump.go

```diff
@@ -83,3 +106,59 @@ func Dump(values ...interface{}) {
 	_, _ = fmt.Fprintf(buff, "\n")
 	fmt.Print(buff.String())
 }
+
+type lineDescriptor struct {
+	targetLine    string
+	dumpVariables []string
+}
+
+type cacheKey struct {
+	file string
+	line int
+}
+
+type cacheStripe struct {
+	cache map[cacheKey]lineDescriptor
+}
+
+type pCache struct {
+	maxSize int
+	pool    *sync.Pool
+}
+
+// newPCache creates pCache with maxSizePerGoroutine.
+func newPCache(maxSizePerGoroutine uint) *pCache {
+	return &pCache{
+		maxSize: int(maxSizePerGoroutine),
```

Comment:
why `uint` to `int` ?

Reply:
to not allow negative numbers as maxSize